truffle/erc721-example/truffle-config.js

```diff
@@ -31,11 +14,20 @@ module.exports = {
       port: 8545,
       network_id: "*" // Match any network id
     },
-    aurora: {
-      provider: () => setupWallet('https://testnet.aurora.dev'),
-      network_id: 0x4e454153,
+    aurora_testnet: {
```

Comment:
if this is renamed, then we need to update docs here https://doc.aurora.dev/interact/truffle